src/Microsoft.IdentityModel.Protocol.Extensions/AudienceValidator.cs

```diff
@@ -31,19 +32,21 @@ internal class AudienceValidator
     {
         public static void Validate(IEnumerable<string> audiences, SecurityToken securityToken, TokenValidationParameters validationParameters)
         {
+            IdentityModelEventSource.Logger.WriteInformation("AudienceValidator.Validate: validating audience in the jwt token");
+
             if (audiences == null)
             {
-                throw new ArgumentNullException("audiences");
+                LogHelper.LogError(string.Format(CultureInfo.InvariantCulture, ErrorMessages.IDX10000, "AudienceValidator.Validate: audiences"), typeof(ArgumentNullException));
```

Comment:
I removed them from other places too. I think having method name would help diagnose better in case of information logs, though hard coding them is not a good idea. I will write a method that fetches the current method name, there is a way to do that in aspnet but not in core right now.